source/leuven/tier2_hardware/thinking_hardware.rst

```diff
@@ -1,7 +1,8 @@
 Thinking7 hardware
 ==================
 
-Thinking is KU Leuven/UHasselt's older Tier-2 cluster. It has thin nodes, large memory nodes, as well as GPGPU nodes.  It is migrated to CentOS 7, so this section will expand over time.
+Thinking is KU Leuven/UHasselt's older Tier-2 cluster. It has thin nodes,
+as well as GPGPU nodes.
 
 
 Login infrastructure
```

Comment:
Hi Geert-Jan, while you're working on this page, could you also remove "<<<<<<< HEAD" in the "Login Infrastructure" section?

Reply:
@MaximeVdB Thanks, some git merge leftover :)